controller/handler/create-trial.js

```diff
@@ -15,9 +15,33 @@ const database = require('../../model');
  */
 function createTrial (request, reply) {
     const trial = database.sequelize.model('trial');
+    const stage = database.sequelize.model('stage');
+    let newTrial = null;
 
-    trial.create(request.payload)
-    .then((newTrial) => {
+    trial.create({
```

Comment:
Great start,
Before it can be merged this will need to be wrapped as a [transaction](http://docs.sequelizejs.com/en/latest/docs/transactions/#unmanaged-transaction-then-callback) to ensure [database consistency](http://www.webopedia.com/TERM/T/two_phase_commit.html).

which would look something like:

``` javascript
...
let transaction = null;

database.sequelize.transaction()
.then((newTransaction) => {
    transaction = newTransaction;
    return trial.create({...}, {transaction})
})
.then(...)
.then(...)
.then(...)
.then(() => {
    return transaction.commit();
})
.then(() => {
     reply.redirect(`/trial/${newTrial.id}`);
})
.catch((err) => {
     transaction.rollback();
     console.error(err);
     reply(boom.badRequest('Invalid Trial'));
})
...
```